src/quacc/runners/ase.py

```diff
@@ -362,3 +369,171 @@ def _set_sella_kwargs(self, optimizer_kwargs: dict[str, Any]) -> None:
 
         if not self.atoms.pbc.any() and "internal" not in optimizer_kwargs:
             optimizer_kwargs["internal"] = True
+
+
+def run_neb(
+    images,
+    relax_cell: bool = False,
+    fmax: float = 0.01,
+    max_steps: int | None = 1000,
+    optimizer: NEBOptimizer | BFGS = NEBOptimizer,
+    optimizer_kwargs: OptimizerKwargs | None = None,
+    run_kwargs: dict[str, Any] | None = None,
+    neb_kwargs: dict[str, Any] | None = None,
+    copy_files: SourceDirectory | dict[SourceDirectory, Filenames] | None = None,
+) -> list[Atoms]:
+    """
+    Run NEB optimization.
+
+    Parameters
+    ----------
+    images : list of Atoms
+        List of images representing the initial path.
+    relax_cell : bool, optional
+        Whether to relax the unit cell shape and volume.
+    fmax : float, optional
+        Tolerance for the force convergence (in eV/A).
+    max_steps : int or None, optional
+        Maximum number of steps to take.
+    optimizer : Optimizer or BFGS, optional
+        Optimizer class to use.
+    optimizer_kwargs : dict or None, optional
+        Dictionary of kwargs for the optimizer.
+    run_kwargs : dict or None, optional
+        Dictionary of kwargs for the run() method of the optimizer.
+    neb_kwargs : dict or None, optional
+        Dictionary of kwargs for the NEB.
+    copy_files : str or dict or None, optional
+        Files to copy before running the calculation.
+
+    Returns
+    -------
+    Optimizer
+        The ASE Optimizer object used in the NEB run.
+    """
+    # Copy atoms so we don't modify it in-place
+    images = copy_atoms(images)
+    neb = NEB(images, **neb_kwargs)
+
+    dir_lists = []
+    # Perform staging operations
+    # this calc_setup function is not suited for multiple Atoms objects
+    for image in images:
+        tmpdir_i, job_results_dir_i = calc_setup(image, copy_files=copy_files)
+        dir_lists.append([tmpdir_i, job_results_dir_i])
+
+    # Set defaults
+    optimizer_kwargs = recursive_dict_merge(
+        {
+            "logfile": "-" if SETTINGS.DEBUG else dir_lists[0][0] / "opt.log",
+            "restart": dir_lists[0][0] / "opt.json",
+        },
+        optimizer_kwargs,
+    )
+    run_kwargs = run_kwargs or {}
+
+    # Check if trajectory kwarg is specified
+    if "trajectory" in optimizer_kwargs:
+        msg = "Quacc does not support setting the `trajectory` kwarg."
+        raise ValueError(msg)
+
+    # Define the Trajectory object
+    traj_file = dir_lists[0][0] / "neb.traj"
+    traj = Trajectory(traj_file, "w", atoms=neb)
+    optimizer_kwargs["trajectory"] = traj
+
+    # Set volume relaxation constraints, if relevant
+    for i in range(len(images)):
+        if relax_cell and images[i].pbc.any():
+            images[i] = FrechetCellFilter(images[i])
```

Comment:
Probably no need to check for PBCs. If the user specifically requests `relax_cell=True`, that's on them.